src/map/lua/lua_baseentity.cpp

```diff
@@ -10001,6 +10001,25 @@ inline int32 CLuaBaseEntity::spawn(lua_State* L)
     return 0;
 }
 
+inline int32 CLuaBaseEntity::despawn(lua_State* L)
+{
+    DSP_DEBUG_BREAK_IF(m_PBaseEntity == nullptr);
+    DSP_DEBUG_BREAK_IF(m_PBaseEntity->objtype != TYPE_MOB);
+
+    CMobEntity* PMob = (CMobEntity*)m_PBaseEntity;
+
+    if (PMob->PAI->IsSpawned())
+    {
+        PMob->PAI->Despawn();
+        return 1;
```

Comment:
you returned 1 but didn't push anything on the stack, that will probably corrupt the stack